mass_mailing_custom_unsubscribe/views/assets.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+<template id="assets_backend"
+          inherit_id="mass_mailing.assets_backend">
+    <xpath expr="//script[@src='/mass_mailing/static/src/js/unsubscribe.js']" position="replace">
```

Comment:
Please add this little comment to explain this weird replacement we're doing.

```suggestion
    <!-- Need to replace the asset because upstream is impossible to extend -->
    <xpath expr="//script[@src='/mass_mailing/static/src/js/unsubscribe.js']" position="replace">
```